document kubectl delete --interactive flag (#41979)
kubernetes/website: content/en/docs/reference/kubectl/kubectl.md

</td>
</tr>

<tr>
<td colspan="2">KUBECTL_INTERACTIVE_DELETE</td>
</tr>
<tr>

> <td></td><td style="line-height: 130%; word-wrap: break-word;">When set to true, --interactive flag in kubectl delete command will be activated and by passing this flag, users can preview and confirm resources before proceeding to delete. 

Reviewer: ```suggestion
<td></td><td style="line-height: 130%; word-wrap: break-word;">When set to true, the --interactive flag in the kubectl delete command will be activated, allowing users to preview and confirm resources before proceeding to delete. 
```

Author: Thanks @chrisnegus for the suggestion. Updated.

> <td></td><td style="line-height: 130%; word-wrap: break-word;">When set to true, the --interactive flag in the kubectl delete command will be activated, allowing users to preview and confirm resources before proceeding to delete by passing this flag.

Reviewer 2: <td></td><td style="line-height: 130%; word-wrap: break-word;">When set to true, the <tt>--interactive</tt> flag in the kubectl delete command will be activated, allowing users to preview and confirm resources before proceeding to delete by passing this flag.
